lib/irb/helper.rb

```diff
@@ -0,0 +1,25 @@
+require_relative "helper/base"
+
+module IRB
+  module Helper
+    @helpers = {}
+
+    class << self
+      attr_reader :helpers
+
+      def register(name, helper_class)
+        @helpers[name] = helper_class
+      end
+
+      def all_helpers_info
+        @helpers.map do |name, helper_class|
+          { display_name: name, description: helper_class.description, category: helper_class.category }
+        end
+      end
+    end
+
+    # Default helpers
+    require_relative "helper/measure"
+    register(:measure, Helper::Measure)
```

Comment:
I think we should add default helpers with `irb/helper` so users/extensions will be able to override them.